lib/sentry/config.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Sentry.Config do
```

Comment:
Have thought about it, but would prevent people updating config at runtime.  I'm not sure on the popularity of doing that.